### PR TITLE
Adding post 1990's Iorter and moving the db unit test

### PR DESCRIPTION
**Pull request by BobBorges**
Here come the last iorter and an update unit test per #294

Also moved the input files for the db unittest in accordance with plans made re #329

### CODE REVIEW

File: scripts/stats-dashboard/mp-coverage.py [hunk 1]
```diff
@@ -95,7 +95,7 @@ def main():
     baseline_df = pd.read_csv("corpus/quality_assessment/baseline_mp_frequencies_per_year/number_mp_in_parliament.csv")
```
Comment (ninpnin) [inline scripts/stats-dashboard/mp-coverage.py]: These should be provided as arguments–we are moving scripts to other locations soon anyway

**Review by ninpnin — CHANGES_REQUESTED** [review]
I think we should remove hard-coded paths and use argparse arguments if possible

**Comment by BobBorges** [comment]
just in the scripts/stats-dashboard/mp-coverage.py, or you have something else in mind? I actually wanted to move almost everything from that stats dashboard process to the pyriksdagen module.

**Comment by ninpnin** [comment]
I thought just here, since we make changes to it now.

BTW there is a merge conflict in  test/pytestconfig.py now, can you fix that?

**Comment by BobBorges** [comment]
OK, yep, I have it open now.

File: scripts/stats-dashboard/mp-coverage.py (hunk 1 above)
Comment (BobBorges) [reply scripts/stats-dashboard/mp-coverage.py]: [7280c5b](https://github.com/welfare-state-analytics/riksdagen-corpus/pull/477/commits/7280c5b50508d9033a1252720825e30166986964)

**Review by ninpnin — APPROVED** [review]

**Pull request merged**